pax/simulation.py

```diff
@@ -257,6 +257,13 @@ def make_pax_event(self):
 
             photon_detection_times = np.array(photon_detection_times)
 
+            # Add double photoelectron emission
+            if len(photon_detection_times):
+                n_dpe = np.random.binomial(len(photon_detection_times), p=self.config['p_double_pe_emision'])
+                if n_dpe:
+                    dpe_times = np.random.choice(photon_detection_times, n_dpe, replace=False)
+                    photon_detection_times = np.concatenate([photon_detection_times, dpe_times])
```

Comment:
Not from this place in the code, though I agree it would be useful to store somewhere